### ISSUE TITLE
Accessibility gaps

### ISSUE THREAD

**Issue by houqp** [houqp/kiorg#53]
From [mwcampbell](https://lobste.rs/~mwcampbell)

> I just tried kiorg on Windows, and yes, it's partially usable with NVDA. I did find at least one "unknown" widget in the main window. Presumably that's a custom egui widget that doesn't yet expose any accessibility info. The tab order also includes things it shouldn't, and the main list view isn't exposed as a high-level widget the way it should be. But the latter isn't yet fully supported in AccessKit either. Regardless, this is progress. Before AccessKit, my default assumption was that an application like this would be completely inaccessible.
